README.md

```diff
@@ -35,7 +42,19 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+var andy = Instructor.findOne({name: "Andy"}, (err, instructor) => {
```

Comment:
👍 Very close!

We want to add the `wish` to `andy`'s list too, so this should all be done inside the callback. And we need to save changes to the containing object:
```js
var andy = Instructor.findOne({name: "Andy"}, (err, instructor) => {
  console.log(andy)
  var wish = new Wishlist_item(description: "Resin Laying Deer Figurine, Gold")
  andy.wishlist_items.push(wish)
  
  andy.save((err, andy_with_wishes) => {
    if(err){
      console.log(err)
    }else{
      console.log(andy_with_wishes)
    }
  })
})
```